queries/big_query.sql

```diff
@@ -0,0 +1,267 @@
+select max(employee_id), avg(position_id), min(store_id)
```

Comment:
Could it be placed on resources folder instead of root?

Reply:
Done.